{"name": "Ручка /api/v1/courier/login", "status": "passed", "description": "Пробуем залогиниться, используя некорректные данные, ожидаем получить ошибку 400 и текст 'Недостаточно данных для входа'", "parameters": [{"name": "login", "value": "None"}, {"name": "password", "value": "'None'"}], "start": 1710617629914, "stop": 1710617629963, "uuid": "9c8448a3-970c-4b26-ad34-340af836bb9a", "historyId": "ff482f7902d5ff98a4ffe78c815c7276", "testCaseId": "14eb7ed67efb59bc0210cf649d3f4709", "fullName": "test_api.tests.test_login_courier.TestLoginCourier#test_login_courier_empty_fields_error", "labels": [{"name": "parentSuite", "value": "test_api.tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "LAPTOP-KS9V29SF"}, {"name": "thread", "value": "5592-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_api.tests.test_login_courier"}]}